roles/template_service_broker/tasks/main.yml

```diff
@@ -2,7 +2,7 @@
 # do any asserts here
 
 - include: install.yml
-  when: template_service_broker_install | default(false) | bool
```

Comment:
Inadvertent change?

Reply:
Yeah. I thought that I had reverted that. Sorry.

Reply:
Fixed now, with the roles/template_service_broker/tasks/install.yml changes that I meant to add but reverted accidentally instead of the roles/template_service_broker/tasks/main.yml changes.